ui/pages/onboarding-flow/create-password/create-password.js

```diff
@@ -324,21 +316,21 @@ export default function CreatePassword({
                 onClick={() => setTermsChecked(!termsChecked)}
                 checked={termsChecked}
               />
-              <Typography
-                variant={TypographyVariant.H5}
-                boxProps={{ marginLeft: 3 }}
-              >
-                {
-                  ///: BEGIN:ONLY_INCLUDE_IN(build-main,build-beta,build-flask)
-                  t('passwordTermsWarning', [createPasswordLink])
-                  ///: END:ONLY_INCLUDE_IN
-                }
-                {
-                  ///: BEGIN:ONLY_INCLUDE_IN(build-mmi)
-                  t('passwordMmiTermsWarning', [createPasswordLink])
-                  ///: END:ONLY_INCLUDE_IN
-                }
-              </Typography>
+              <Text variant={TextVariant.bodyMd} marginLeft={3}>
+                {t('passwordTermsWarning', [
+                  <a
+                    onClick={(e) => e.stopPropagation()}
+                    key="create-password__link-text"
+                    href={ZENDESK_URLS.PASSWORD_AND_SRP_ARTICLE}
+                    target="_blank"
+                    rel="noopener noreferrer"
+                  >
+                    <span className="create-password__link-text">
+                      {t('learnMoreUpperCase')}
+                    </span>
+                  </a>,
+                ])}
```

Comment:
Seems to be some missing build type comment logic here too
```suggestion
                {
                  ///: BEGIN:ONLY_INCLUDE_IN(build-main,build-beta,build-flask)
                  t('passwordTermsWarning', [createPasswordLink])
                  ///: END:ONLY_INCLUDE_IN
                }
                {
                  ///: BEGIN:ONLY_INCLUDE_IN(build-mmi)
                  t('passwordMmiTermsWarning', [createPasswordLink])
                  ///: END:ONLY_INCLUDE_IN
                }
```